source/ftrack_api/_version.py

```diff
@@ -1,2 +1,2 @@
 # These version placeholders will be replaced later during substitution.
-__version__ = "0.0.1.dev"
+__version__ = "3.0.0rc1.post1.dev0+3a424f3"
```

Comment:
I don't think changes to `_version.py` should be committed, or I am missing something here.